src/state/slices/tradeQuoteSlice/tradeQuoteSlice.ts

```diff
@@ -13,18 +15,37 @@ import {
 
 export type TradeQuoteSliceState = {
   activeStep: number | undefined // Make sure to actively check for undefined vs. falsy here. 0 is the first step, undefined means no active step yet
-  activeQuoteIndex: number | undefined // the selected swapper used to find the active quote in the api response
+  activeQuoteId: { swapperName: SwapperName; quoteId: string } | undefined // the selected quote id used to find the active quote in the api responses
```

Comment:
This isn't an ID anymore 👀

Reply:
gm 25ab897ca0041e9ebb4424e19954dac4768b7b0c